website/singlepages/templates/singlepages/trust_contact_persons.html

```diff
@@ -0,0 +1,62 @@
+{% extends "base.html" %}
+{% load i18n static grid_item %}
+
+{% block title %}{% trans "Contact persons" %} — {{ block.super }}{% endblock %}
+{% block opengraph_title %}{% trans "Contact persons" %} — {{ block.super }}{% endblock %}
+
+{% block body %}
+    <section class="page-section">
+        <div class="container">
+            <h1 class="section-title">{% trans "Trust contact persons" %}</h1>
+            {% blocktrans %}
+                If you struggle with anything from personal problems to questions about
+                your study and you don’t know where to go, you can contact one of the
+                trust contact persons. They will make sure you get in contact with the
+                right professional. Everything that is discussed with a trust contact
+                person is confidential.
+            {% endblocktrans %}
+            <div class="row mt-4">
+                <div class="col-6 my-3">
+                    {% static "img/contact_persons/tom-koning.jpeg" as photo %}
+                    {% grid_item title="Tom Koning" meta_text="<p>gtkoning.tk@gmail.com</p>" url="mailto:gtkoning.tk@gmail.com" image_url=photo class_name="ratio2-3" %}
+                </div>
+                <div class="col-6 my-3">
+                    {% static "img/contact_persons/isolde-hoogendonk.jpg" as photo %}
+                    {% grid_item title="Isolde Hoogendonk" meta_text="<p>isoldeedlosi@gmail.com</p>" url="mailto:isoldeedlosi@gmail.com" image_url=photo class_name="ratio2-3" %}
```

Comment:
Are we putting these email addresses in our code? Not a setting or something?

Reply:
And don't we just want these images to be square like the rest of the website?

Reply:
I figured to make it as 'dumb' as possible. Making it a setting would also be possible, but then we could also just make it a true app

Reply:
I think I just missed the ending discussion part about it maybe. We have a template tag that directly outputs stuff from the settings. Seems easy to do, but I get your point. We have hardcoded email addresses in more places anyways.